inventories/nos-social/inventory.yml

```diff
@@ -0,0 +1,16 @@
+---
+nos-social:
```

Comment:
This group name may cause weird errors.  Python(and thus Ansible) want snake_case and I find having inventory groups with a dash instead of underscore can cause issues.  I recommend changing our instances of `nos-social` to `nos_social` within this file, along with the name of the inventory, role, and playbook.